tools/decode-status.py

```diff
@@ -236,7 +236,7 @@
     "USE_BH1750","USE_VEML6070","USE_ADS1115_I2CDEV","USE_ADS1115",
     "USE_INA219","USE_SHT3X","USE_MHZ19","USE_TSL2561",
     "USE_SENSEAIR","USE_PMS5003","USE_MGS","USE_NOVA_SDS",
-    "USE_SGP30","USE_SR04","USE_SDM120","USE_SI1145",
+    "USE_SGP30","USE_SGP40","USE_SR04","USE_SDM120","USE_SI1145",
     "USE_SDM630","USE_LM75AD","USE_APDS9960","USE_TM1638"
```

Comment:
You're not supposed to insert ANYTHING here which does not relate to the correct location in file ``support_features.ino``.

In your case you'll need to move the entry down into the new array following USE_DISPLAY_TM1621_SONOFF